evalml/data_checks/datetime_format_data_check.py

```diff
@@ -42,8 +42,8 @@ def validate(self, X, y):
             The column "dates" has a set of dates with hourly frequency appended to the end of a series of days, which is inconsistent
             with the frequency of the previous 9 dates (1 day).
 
-            >>> X = pd.DataFrame(pd.date_range("2021-01-01", periods=6).append(pd.date_range("2021-01-07", periods=3, freq="H")), columns=["dates"])
-            >>> y = pd.Series([0, 1, 0, 1, 1, 0, 0, 0, 1, 0])
+            >>> X = pd.DataFrame(pd.date_range("2015-01-01", periods=2).append(pd.date_range("2015-01-08", periods=2, freq="H").append(pd.date_range("2016-03-02", periods=2, freq="M").append(pd.date_range("2017-05-21", periods=2, freq="5D")))), columns=["dates"])
```

Comment:
Can you update the docstring description above on lines 42-43 to reflect the new data being used in this test?